src/main/kotlin/no/nav/syfo/service/DialogmoteInnkallingNarmesteLederVarselService.kt

```diff
@@ -0,0 +1,328 @@
+            return
+        }
+
+        if (narmesteLederRelasjon.narmesteLederEpost == null) {
+            log.error("Sender ikke varsel: narmesteLederRelasjon mangler epost")
+            return
+        }
+
+        val lenkeTilDialogmoteLanding = "$dialogmoterUrl/arbeidsgiver/${narmesteLederRelasjon.narmesteLederId}"
+        val personData = pdlClient.hentPerson(personIdent = varselHendelse.arbeidstakerFnr)
+
+        sendInfoTilNarmesteLeder(
+            varselHendelse = varselHendelse,
+            personData = personData,
+            lenkeTilDialogmoteLanding = lenkeTilDialogmoteLanding,
+            ledersEpost = narmesteLederRelasjon.narmesteLederEpost,
+            narmesteLederId = narmesteLederRelasjon.narmesteLederId
+        )
+    }
+
+    private suspend fun sendInfoTilNarmesteLeder(
+        varselHendelse: NarmesteLederHendelse,
+        personData: HentPersonData?,
+        lenkeTilDialogmoteLanding: String,
+        ledersEpost: String,
+        narmesteLederId: String,
+    ) {
+        when (varselHendelse.type) {
+            NL_DIALOGMOTE_INNKALT -> handleInnkalt(
+                varselHendelse = varselHendelse,
+                personData = personData,
+                lenkeTilDialogmoteLanding = lenkeTilDialogmoteLanding,
+                ledersEpost = ledersEpost,
+                narmesteLederId = narmesteLederId
+            )
+
+            NL_DIALOGMOTE_NYTT_TID_STED -> handleNyttTidSted(
+                varselHendelse = varselHendelse,
+                personData = personData,
+                lenkeTilDialogmoteLanding = lenkeTilDialogmoteLanding,
+                ledersEpost = ledersEpost,
+                narmesteLederId = narmesteLederId
+            )
+
+            NL_DIALOGMOTE_AVLYST -> handleAvlyst(
+                varselHendelse = varselHendelse,
+                personData = personData,
+                ledersEpost = ledersEpost,
+                narmesteLederId = narmesteLederId
+            )
+
+            NL_DIALOGMOTE_REFERAT -> handleReferat(
+                varselHendelse = varselHendelse,
+                narmesteLederId = narmesteLederId
+            )
+
+            NL_DIALOGMOTE_SVAR -> handleSvar(
+                varselHendelse = varselHendelse,
+                narmesteLederId = narmesteLederId
+            )
+
+            else -> log.error("Forsøkte å sende nærmeste leder dialogmøtevarsel for type: ${varselHendelse.type.name}")
+        }
+    }
+
+    private suspend fun handleInnkalt(
+        varselHendelse: NarmesteLederHendelse,
+        personData: HentPersonData?,
+        lenkeTilDialogmoteLanding: String,
+        ledersEpost: String,
+        narmesteLederId: String,
+    ) {
+        sendVarselTilDineSykmeldte(varselHendelse)
+        val motetidspunkt = varselHendelse.data?.toVarselData()?.motetidspunkt?.tidspunkt
+        require(motetidspunkt != null) { "Dialogmøte-innkalling mangler motetidspunkt" }
```

Comment:
Kan vi kanskje sjekke også at møtetidspunkt er i framtiden? Eventuelt lage møtetidspunkt validering funksjon og egen exception ala  DialogmotetidspunktValidationException("msg for avery case")

Reply:
@Vitenok @MalazAlkoj Kan godt legge det til om dere vil, men det er ikke funksjonelt mulig i Modia å kalle inn i fortiden. Tenker dere likevel vi bør ha det?

Reply:
Om vi vil kjøre en zero-trust konsept, kan det være greit å fortsatt sjekke.

Reply:
Ok! Legger til :)